lib/y2_installbase.pm

```diff
@@ -179,6 +179,7 @@ sub process_patterns {
     if (get_required_var('PATTERNS')) {
         if (check_var('PATTERNS', 'all') && !check_var('VIDEOMODE', 'text')) {
             $self->select_all_patterns_by_menu();
+            $self->deselect_pattern() if get_var('DEPATTERNS');
```

Comment:
i suggest use EXCLUDE_PATTERNS

Reply:
Thanks, this setting name looks better. I have updated code and PR title and commit.